var/spack/repos/builtin/packages/fftw/package.py

```diff
@@ -115,6 +115,13 @@ def configure(self, spec, prefix):
             float_options.append('--enable-sse2')
             double_options.append('--enable-sse2')
 
+        # Support for Intel compiler:
+        # https://software.intel.com/en-us/articles/performance-tools-for-software-developers-building-fftw-with-the-intel-compilers
+        if '%intel' in spec:
+            options.append(
+                "CFLAGS=-gcc --with-our-malloc16 --with-combined-threads"
```

Comment:
I have a couple of concerns about this. The first one is that the current version of `fftw` configure help says:
```console
  ...
  --with-our-malloc       use our aligned malloc (helpful for Win32)
  --with-our-malloc16     Obsolete alias for --with-our-malloc16
  ...
```
while the page you linked says:
> This guide is intended to help Intel® C++ Compiler customers build the FFTW* library on Mac OS*

I think we should at least check what is the OS we are running on before appending these options. The second one is that the page dates December 3, 2008: are these suggestions still relevant?